spec/models/reservation_spec.rb

```diff
@@ -99,7 +99,19 @@
     it { is_expected.to respond_to(:find_renewal_date) }
   end
 
-  context 'when not checked out' do
+  context 'when not checked out and not upcoming' do
+    subject(:reservation) do
+      FactoryGirl.build(:valid_reservation,
+                        start_date: Time.zone.today + 1)
+    end
+    it { expect(reservation.status).to eq('reserved') }
+    # currently returns true; doesn't check for checked out
+    it { expect(reservation).to_not be_eligible_for_renew }
+  end
+
+  context 'when upcoming' do
```

Comment:
this is probably leftover from when I was editing the status method to get 'due today' and 'starts today' emails; it shouldn't be necessary now, I'll just checkout the reservation spec from master